manifests/plugin/python.pp

```diff
@@ -11,12 +11,6 @@
 
   validate_hash($config)
 
-  if $::osfamily == 'Redhat' {
-    package { 'collectd-python':
-      ensure => $ensure,
-    }
-  }
```

Comment:
Removed from EPEL7? If so then add some logic so it still works on EPEL6 and EPEL5